roles/nginx/tasks/ssl.yml

```diff
@@ -37,4 +37,4 @@
     minute: 1
     hour: 23
     weekday: 0
-    job: '/usr/bin/certbot renew --quiet --renew-hook "/usr/sbin/service nginx reload"'
\ No newline at end of file
+    job: '/usr/bin/certbot renew --quiet --renew-hook "/usr/sbin/service nginx reload"'
```

Comment:
It was already there but I'm not certain this work. Have you tried it @mrasband ?

I usually use `--pre-hook "systemctl stop nginx.service" --post-hook "systemctl start nginx.service"`